plugins/html-plugin/src/main/java/ru/sbtqa/tag/pagefactory/utils/HtmlElementUtils.java

```diff
@@ -0,0 +1,51 @@
+package ru.sbtqa.tag.pagefactory.utils;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.PageFactory;
+
+import ru.sbtqa.tag.pagefactory.context.PageContext;
+import ru.sbtqa.tag.pagefactory.environment.Environment;
+import ru.sbtqa.tag.pagefactory.html.loader.CustomHtmlElementLoader;
+import ru.sbtqa.tag.pagefactory.html.loader.decorators.CustomHtmlElementDecorator;
+import ru.yandex.qatools.htmlelements.element.TypifiedElement;
+import ru.yandex.qatools.htmlelements.loader.decorator.HtmlElementLocatorFactory;
+import static ru.yandex.qatools.htmlelements.utils.HtmlElementUtils.isTypifiedElement;
+
+public class HtmlElementUtils {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a private constructor to hide the implicit public one. ([squid:S1118](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS1118))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=133&issues=AWfPt2pB4tUZZzFFj6Hc&open=AWfPt2pB4tUZZzFFj6Hc)

[ISSUE_KEY]: AWfPt2pB4tUZZzFFj6Hc